azuredevops/internal/acceptancetests/resource_serviceendpoint_kubernetes_test.go

```diff
@@ -21,6 +21,7 @@ const terraformServiceEndpointNode = "azuredevops_serviceendpoint_kubernetes.ser
 // validates that an apply followed by another apply (i.e., resource update) will be reflected in AzDO and the
 // underlying terraform state.
 func TestAccServiceEndpointKubernetesForAzureSubscriptionCreateAndUpdate(t *testing.T) {
+	t.Skip("Skipping test TestAccServiceEndpointKubernetesForAzureSubscriptionCreateAndUpdate: test resource limit")
```

Comment:
Is this a new issue?

Reply:
I add state check during creation,   dummy value won't be able to create the real service connection though  we can find it in the `service connection` page.  I disable it due to test resource and sensitive configurations.